TestsFoundation/TestsFoundation/Element/K5NavigationBar.swift

```diff
@@ -19,9 +19,17 @@
 import Foundation
 
 public enum K5NavigationBar: String, ElementWrapper {
-    case schedule, grades, resources
+    case resources
 
     public static var homeroom: Element {
         return app.find(label: "Homeroom")
     }
+
+    public static var schedule: Element {
+        return app.find(label: "Schedule")
+    }
+
+    public static var grades: Element {
+        return app.find(label: "Grades")
```

Comment:
In these cases the `return` statement can be omitted. Sometimes we even make the `var` a one-liner if it's short.